style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 48px;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: flex-start;
+  padding: 8px;
+  height: 40px;
+  width: 100%;
+  left: 311px;
+  top: 4px;
+  border-radius: 8px;
+}
+
+.menu-icon {
+  padding-right: 30px;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('/images/header-illsutration-mobile.svg');
```

Comment:
**OPTIONAL** - Just as I said above, please fix the issue with the image navigation 👍 👍